arm-commerce/2015-06-01-preview/swagger/commerce.json

```diff
@@ -114,7 +114,7 @@
           {
             "name": "$filter",
             "in": "query",
-            "required": false,
+            "required": true,
```

Comment:
Is the api-version "2015-06-01-preview" still the same? When are you guys planning to move out of preview?